zingolib/src/lightclient/sync.rs

```diff
@@ -681,11 +681,12 @@ pub mod test {
         std::env::set_var("RUST_BACKTRACE", "1");
         let wallet = wallet_case.load_example_wallet().await;
         let lc = LightClient::create_from_wallet_async(wallet).await.unwrap();
-        let _ = lc.do_sync(true).await;
+        lc.do_sync(true).await.unwrap();
```

Comment:
May as well make it an `expect("If sync panics here, then the test is already interesting.")`